src/univariate/continuous/normal.jl

```diff
@@ -35,6 +35,11 @@ entropy(d::Normal) = 0.5 * (log2π + 1.0) + log(d.σ)
 
 @_delegate_statsfuns Normal norm μ σ
 
+### Added by scidom: start
+logpdf(d::Normal, x::Real) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
+logpdf(d::Normal, x::Number) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
```

Comment:
Probably not. The promotions should really happen at the distribution level if necessary